packages/xonotic/xonotic.pacscript

```diff
@@ -0,0 +1,21 @@
+name="xonotic"
+pkgname="xonotic"
+pkgver="0.8.6"
+gives="xonotic"
+pkgdesc="The Free and Fast Arena Shooter"
+url="https://dl.unvanquished.net/share/xonotic/release/xonotic-${pkgver}.zip"
+hash="50850f8d800e7499722f6ea61e478e96464a375494b5a24da93aa0598cbe964d"
+breaks=("xonotic-git")
+arch=('amd64')
+maintainer="Herisson <spaceguybox@outlook.com>"
+repology=("project: xonotic")
+
+package() {
+  sudo install -Dm755 "xonotic-linux64-sdl" "${pkgdir}/usr/bin/${pkgname}-sdl"
+  sudo install -Dm755 "xonotic-linux64-glx" "${pkgdir}/usr/bin/${pkgname}-glx"
+  sudo mkdir -p "${pkgdir}/usr/share/applications/"
+  sudo mkdir -p "~/.xonotic"
```

Comment:
Can we not install anywhere else? Also use `${homedir}` instead of `~`

Reply:
no we can not, else xonotic does not launch and cannot find the data directory. we could in theory redirect it to another path and add a flag to move it but that requires us to write a new .desktop file entirely

Reply:
Let's do that, because application data should never be in the home directory.

Reply:
okay, it'll be way more annoying but sure.

Reply:
@Elsie19 https://xonotic.org/faq/#config it wants it there for the configs though too

Reply:
we could force -basedir for the game data but the configs may not be movable

Reply:
That's good. Configs are ok to be left in the homedir, it's just I'd really prefer the app data not to be.

Reply:
then we need to make a fresh .desktop with the exec as ```xonotic -basedir /somedir````

Reply:
https://github.com/pacstall/pacstall/wiki/Pacscript-101#other

Check here for how to add a desktop file.

Reply:
thank you